Pipfile.lock

```diff
@@ -1184,11 +1252,11 @@
         },
         "sphinx-rtd-theme": {
             "hashes": [
-                "sha256:32bd3b5d13dc8186d7a42fc816a23d32e83a4827d7d9882948e7b837c232da5a",
-                "sha256:4a05bdbe8b1446d77a01e20a23ebc6777c74f43237035e76be89699308987d6f"
+                "sha256:4d35a56f4508cfee4c4fb604373ede6feae2a306731d533f409ef5c3496fdbd8",
+                "sha256:eec6d497e4c2195fa0e8b2016b337532b8a699a68bcb22a512870e16925c6a5c"
             ],
             "index": "pypi",
-            "version": "==0.5.2"
+            "version": "==1.0.0"
         },
```

Comment:
One quick fix for the documentation build would be to restore the old version here manually.
Or you could find out what in the footer termplate changed and fix the patch file...

Reply:
Thank you, that sounds good to me.
It was not even my intention to update so many parts of this lock file,
but when I added a dependency to setup.py and ran `pipenv install -e .` like our docu says,
I expected `django-webpack-loader` to show up in the pipfile.lock, but it was not listed in the lock file.

So I applied this patch: I ran `pipenv install -e .` followed by `pipenv lock` to move the newly installed `django-webpack-loader` into the pifile.lock, but this has the drawback, that other unnecessary entries in the pipfile.lock updated, too.

Do you know a better approach for the pipfile.lock sync?
Maybe we could try this tool [pipenv-setup](https://github.com/pypa/pipenv/issues/1263#issuecomment-524382945)?

Reply:
Yes, this would be a great solution and probably also fix the docu generation problem, but are you sure that `django-webpack-loader` is a dev-dependency? I first thought we need this django app also installed in production, because It provides template tags like `{% render_bundle some_webpack_entry_point %}` which it is basically used for creating the script tag for webpack build output (replacing the ¸<script src="{% static 'main.js' %}" >) and not part of the webpack build process.

So we need this in production too or am I wrong?

Reply:
Ah okay sorry, I wasn't aware how the plugin works. But yes, should be a production dependency...
So it's probably best to revert the sphinx theme version manually...

Reply:
You are right, reverting the sphinx theme version worked great to fix the docu generation, thank you :tada: 
Just some pylint and test issues remain, I will have a look at them